fundamentals/authors.tex

```diff
@@ -6,7 +6,5 @@
   Matthew J. Gidden$^2$,\\
   Meghan B. McGarry$^2$,\\
   Arrielle C. Opotowsky$^2$,\\
-  Olzhas Rakhimov$^2$,\\
-  Zach Welch$^2$,\\
```

Comment:
See #45 and #43 : Months ago, I requested contributions. Minus those, I made another request, specifically requesting contributions by Friday. At that time, I expressed that they would be moved to the acknowledgements otherwise. Without their approval of the paper, it's not ethical for me to include them as authors. Without their contributions, I think the acknowledgements are appropriate.